### PR TITLE
Add foreman azure rm 2.0.0.pre1

### PR DESCRIPTION
**Pull request by apuntamb**
Supported Versions:

 * Nightly
 * 1.23
 * 1.22
 * 1.21

### CODE REVIEW

File: packages/plugins/rubygem-foreman_azure_rm/rubygem-foreman_azure_rm.spec [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+#   2. Delete these lines
+#
+# Generated from foreman_azure_rm-2.0.0.pre1.gem by gem2rpm -*- rpm-spec -*-
+# template: foreman_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_azure_rm
+%global plugin_name azure_rm
+%global foreman_min_version FIXME
```
Comment (ekohl) [inline packages/plugins/rubygem-foreman_azure_rm/rubygem-foreman_azure_rm.spec]: Could you fill this in and then remove the top 4 lines?

File: packages/plugins/rubygem-foreman_azure_rm/rubygem-foreman_azure_rm.spec [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 2.0.0.pre1
+Release: 1%{?foremandist}%{?dist}
+Summary: Azure Resource Manager as a compute resource for The Foreman
+Group: Applications/Systems
+License: GPL-3.0
```
Comment (ekohl) [inline packages/plugins/rubygem-foreman_azure_rm/rubygem-foreman_azure_rm.spec]: Open in issue in our tooling, but the format gems and RPMs use differs:
```suggestion
License: GPLv3
```

File: packages/plugins/rubygem-foreman_azure_rm/rubygem-foreman_azure_rm.spec (hunk 1 above)
Comment (apuntamb) [reply packages/plugins/rubygem-foreman_azure_rm/rubygem-foreman_azure_rm.spec]: done!

File: packages/plugins/rubygem-foreman_azure_rm/rubygem-foreman_azure_rm.spec (hunk 2 above)
Comment (apuntamb) [reply packages/plugins/rubygem-foreman_azure_rm/rubygem-foreman_azure_rm.spec]: updated.

**Comment by ekohl** [comment]
[test rpm] looks like the job was killed for some reason

**Comment by apuntamb** [comment]
> [test rpm] looks like the job was killed for some reason

As per the logs, it seems the `git annex whereis` command shows failure. I'm not sure why is this caused. What do you think could be the solution to this?

**Comment by ShimShtein** [comment]
[test rpm]

**Comment by ekohl** [comment]
Looks like the build is getting killed after 2 hours because that's the maximum time we allow for such a job. I wonder why it's taking that long.

**Comment by ShimShtein** [comment]
No idea, and don't see anything in Jenkins log. Are there additional files to look into?

**Comment by ekohl** [comment]
I've tried to look into it, but I can't really explain it. Maybe it's because of their size and the amount of files. Looks like the SDKs have multiple versions in them making them quite large. RPMs do post processing to see which dependencies are there and that may be slow.

**Comment by apuntamb** [comment]
> I've tried to look into it, but I can't really explain it. Maybe it's because of their size and the amount of files. Looks like the SDKs have multiple versions in them making them quite large. RPMs do post processing to see which dependencies are there and that may be slow.

Okk, yep they do have some multiple versions. Then how do you suggest to go about it?

**Comment by ekohl** [comment]
It's probably best to split up the PR into multiple smaller PRs to work around it. I can't think of a better workaround.

**Comment by evgeni** [comment]
@apuntamb can you please rebase this ontop of `rpm/develop` now that #4179 is merged?

**Comment by evgeni** [comment]
obsoleted by https://github.com/theforeman/foreman-packaging/pull/4257

**Pull request closed without merging**